.npmignore

```diff
@@ -1,3 +1,2 @@
-src/
```

Comment:
Ideally, to reduce bundle size & install time it's best practice to run the typescript build on the development machine rather than have the typescript source be downloaded and compiled upon install with npm.

Could we please change this back?